### PR TITLE
improve HPE ProCurve & Aruba Switch Series

### PR DESCRIPTION
**Pull request by robinreinhardt**

### CODE REVIEW

File: device-types/HPE/ProCurve-2810-24G.yaml [hunk 1]
```diff
@@ -53,11 +58,19 @@ interfaces:
     type: 1000base-t
   - name: '20'
     type: 1000base-t
-  - name: '21'
+  - name: 21-RJ45
```
Comment (danner26) [inline device-types/HPE/ProCurve-2810-24G.yaml:61]: Are these interface names in the config actually suffixed with `-RJ45`? Can you show us an example?
If not, please revert this change and adjust the SFP ones accordingly.

File: device-types/HPE/ProCurve-2810-48G.yaml [hunk 2]
```diff
@@ -101,11 +106,19 @@ interfaces:
     type: 1000base-t
   - name: '44'
     type: 1000base-t
-  - name: '45'
+  - name: 45-RJ45
```
Comment (danner26) [inline device-types/HPE/ProCurve-2810-48G.yaml:109]: Are these interface names in the config actually suffixed with `-RJ45`? Can you show us an example?
If not, please revert this change and adjust the SFP ones accordingly.

**Review by danner26 — CHANGES_REQUESTED** [review]
Please review my questions, and as annoying as this might be can you please remove the extra white space around all of the images? Please crop them so that there is not any extra space. Thank you.

File: device-types/HPE/ProCurve-2810-24G.yaml (hunk 1 above)
Comment (robinreinhardt) [reply device-types/HPE/ProCurve-2810-24G.yaml:61]: They are not. They are labeled "T" and "S" on the chassis, but there is only one interface in the software that does not have this suffix.
```
2530-8G-PoEP# show int brief

 Status and Counters - Port Status

                   | Intrusion                           MDI  Flow
  Port  Type       | Alert     Enabled Status Mode       Mode Ctrl
  ----- ---------- + --------- ------- ------ ---------- ---- ----
  1     100/1000T  | No        Yes     Down   1000FDx    MDI  off 
  2     100/1000T  | No        Yes     Down   1000FDx    MDIX off 
  3     100/1000T  | No        Yes     Down   1000FDx    MDI  off 
  4     100/1000T  | No        Yes     Down   1000FDx    MDI  off 
  5     100/1000T  | No        Yes     Down   1000FDx    MDIX off 
  6     100/1000T  | No        Yes     Down   1000FDx    MDIX off 
  7     100/1000T  | No        Yes     Down   1000FDx    MDIX off 
  8     100/1000T  | No        Yes     Down   1000FDx    MDIX off 
  9     100/1000T  | No        Yes     Down   1000FDx    MDI  off 
  10    1000LX     | No        Yes     Up     1000FDx    NA   off
```

Thats why I used other Aruba switches as a reference and adopted the solution from the Aruba 2530 8G.
I'm not quiet sure how to define this accurately.

Comment (danner26) [reply device-types/HPE/ProCurve-2810-24G.yaml:61]: Yeah they must not be defined correctly then. Can you show me the config for the ports in question here? 21, 22, etc

Comment (robinreinhardt) [reply device-types/HPE/ProCurve-2810-24G.yaml:61]: Okay. Sure, here is the output:

```
2810-24G# show int brief

 Status and Counters - Port Status

                  | Intrusion                           MDI   Flow  Bcast 
  Port  Type      | Alert     Enabled Status Mode       Mode  Ctrl  Limit 
  ----- --------- + --------- ------- ------ ---------- ----- ----- ------
  1     100/1000T | No        Yes     Up     1000FDx    MDIX  off   0     
  2     100/1000T | No        Yes     Up     1000FDx    MDI   off   0     
  3     100/1000T | No        Yes     Up     1000FDx    MDI   off   0     
  4     100/1000T | No        Yes     Down   1000FDx    MDIX  off   0     
  ...
  ...
  21    100/1000T | No        Yes     Down   1000FDx    MDI   off   0     
  22    100/1000T | No        Yes     Down   1000FDx    MDI   off   0     
  23    100/1000T | No        Yes     Up     1000FDx    MDIX  off   0     
  24    100/1000T | No        Yes     Down   1000FDx    MDI   off   0
```

Comment (danner26) [reply device-types/HPE/ProCurve-2810-24G.yaml:61]: Yeah so the port names are just `21`, `22`, etc

Comment (robinreinhardt) [reply device-types/HPE/ProCurve-2810-24G.yaml]: okay, fixed it

**Review by danner26 — APPROVED** [review]

**Comment by robinreinhardt** [comment]
merged and fixed the amount of management modules (unlike the 5400R models, the 5400zl can only have one) and PSU naming

**Review by danner26 — APPROVED** [review]
